src/main/java/org/humancellatlas/ingest/project/web/ProjectController.java

```diff
@@ -44,15 +50,33 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(ProjectController.class);
 
     private final @NonNull ProjectService projectService;
+    private final @NonNull ProjectEventHandler projectEventHandler;
+
     private final @NonNull PagedResourcesAssembler pagedResourcesAssembler;
 
+    private final @NonNull ProjectRepository projectRepository;
+
+    private final @NonNull JsonPatcher jsonPatcher;
+
     @PostMapping("/projects")
-    ResponseEntity<Resource> register(@RequestBody final Project project,
+    ResponseEntity<Resource<?>> register(@RequestBody final Project project,
             final PersistentEntityResourceAssembler assembler) {
         Project result = projectService.register(project);
         return ResponseEntity.ok().body(assembler.toFullResource(result));
     }
 
+    @PatchMapping("/projects/{id}")
+    ResponseEntity<Resource<?>> update(@PathVariable("id") final Project project,
```

Comment:
It looks like this API change doesn't warrant a major version bump to the `/projects` API. @rdgoite Could you confirm this in an end-to-end test?

Reply:
I'm not sure I understand "major version bump" in this context. This is a minor change in the API to add partial update flagging.

Reply:
I agree - this looks like a minor change. Have you confirmed so in an end-to-end test?

Reply:
There's an integration test that's part of this PR, and that seems to have passed.

Reply:
I think a contract test would be more suitable for asserting that this change works with other components. In absence of that, the end-to-end would also suffice